### DISCUSSION TITLE
Mobile App (Android, iPhone)

### DISCUSSION THREAD

**Discussion by Kazz3r24** [sct/overseerr#2376] [Feature Requests]
I just switched over from Ombi to check out Overseerr, and so far I'm really liking it!
I was wondering if there's a plan at some point to have a mobile app that can be used to make/approve/deny requests? I ask because Ombi has one and it's a night and day difference when it comes to being an admin for a media server.

Even if the webpage was written to be responsive on mobile for webpages, it's still not the same in my opinion.

I just love being able to quickly approve requests from the comfort of my couch.

**Comment by TheCatLady** [comment]
No plans to develop a mobile app, since Overseerr functions wonderfully as a PWA.
